provider/machine_pool_resource.go

```diff
@@ -184,6 +184,16 @@ func (t *MachinePoolResourceType) GetSchema(ctx context.Context) (result tfsdk.S
 					ValueCannotBeChangedModifier(),
 				},
 			},
+			"disk_size": {
```

Comment:
Please generate the attributes in the documentation

Reply:
done

Reply:
regenerate it please

Reply:
done